### DISCUSSION TITLE
Performance degradation when managing large numbers of ICONS

### DISCUSSION THREAD

**Discussion by soultipsy** [uxbox/uxbox#1009] [Q&A] [answered]
When I try to manage a large number of ICONS in a project, there is a significant performance degradation. When the mouse hover icon takes a long time to show its focus.  When a icon is selected, other ICONS will not be selected for quite a long time, and the wrong icon position will be dragged.
I don't think this is a bug, but please try to fix it to improve the performance of the software.

https://user-images.githubusercontent.com/40300919/120570702-425e4e00-c44b-11eb-9b50-ceefc992a122.mp4

**Comment by myfunnyandy** [comment, marked as answer]
Hi @soultipsy, we regulary spend time on improvements related to performance. This type of use cases are really valuable for us as they help us tackle specific issues, we will take it into account.

Thanks!
